core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiService.scala

```diff
@@ -145,13 +145,6 @@ trait AdminApiService extends HttpService with PerRequestCreator with UserInfoDi
           UserService.AdminSynchronizeGroupMembers(RawlsGroupRef(RawlsGroupName(groupName))))
       }
     } ~
-    path("admin" / "users") {
```

Comment:
So we're dropping this from the web service API? It's admin so I guess we should be the only ones depending on it anyway, and I guess it probably belongs in Sam instead of Rawls. I guess I'm okay with this. Need to remove it from `rawls.yaml`.